assume/strategies/advanced_orders.py

```diff
@@ -166,20 +166,21 @@ def calculate_bids(
         for i in range(len(bid_price_block)):
             price += bid_price_block[i] * list(bid_quantity_block.values())[i]
             volume += list(bid_quantity_block.values())[i]
-        mean_price = price / volume
+        if volume != 0:
+            mean_price = price / volume
 
-        bids.append(
-            {
-                "start_time": product_tuples[0][0],
-                "end_time": product_tuples[-1][1],
-                "only_hours": product_tuples[0][2],
-                "price": mean_price,
-                "volume": bid_quantity_block,
-                "bid_type": "BB",
-                "min_acceptance_ratio": 1,
-                "accepted_volume": {product[0]: 0 for product in product_tuples},
-            }
-        )
+            bids.append(
+                {
+                    "start_time": product_tuples[0][0],
+                    "end_time": product_tuples[-1][1],
+                    "only_hours": product_tuples[0][2],
+                    "price": mean_price,
+                    "volume": bid_quantity_block,
+                    "bid_type": "BB",
+                    "min_acceptance_ratio": 1,
+                    "accepted_volume": {product[0]: 0 for product in product_tuples},
+                }
+            )
 
         bids = self.remove_empty_bids(bids)
```

Comment:
since there is already a check for volume != 0 this is not needed

Reply:
We only check the volume for the block order, so we need this here.